### ISSUE TITLE
Training stuck for `text_classification_from_scratch` example

### ISSUE THREAD

**Issue by fchollet** [keras-team/keras-core#224]
First non-trivial issue encountered in keras.io example conversion: `text_classification_from_scratch` doesn't train. Loss / accuracy can't get above random baseline. It reliably trains in tf.keras.

Main suspect: embedding layer?
